.travis.yml

```diff
@@ -6,12 +6,18 @@ android:
     - tools
     - build-tools-28.0.3
     - android-28
+    - android-30
+    - sys-img-x86-android-27
 before_install:
  - yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-28"
+ - emulator -avd test -no-audio -no-window &
```

Comment:
Please move 13, 14 lines below 16 line as currently, this is trying to initiate the emulator before creating the virtual environment

Reply:
Done